config/index.js

```diff
@@ -477,6 +477,13 @@ var conf = convict({
       doc: 'enable security history',
       default: true,
       env: 'SECURITY_HISTORY_ENABLED'
+    },
+    ipProfiling: {
+      enabled: {
+        doc: 'enable ip profiling',
```

Comment:
nit: this docstring should probably say something about what effects the profiling will have.